docs/component-implementation-1.md

```diff
@@ -294,6 +294,15 @@ Finally, add the `BlinkIntervalSet` event with an argument of `U32` type to indi
 
 > Resolve any `fprime-util build` errors before continuing 
 
+## Try it yourself
```

Comment:
I had intended that the paragraph on line `293` be a "Try it yourself" table too... as that is the paragraph that is missed.  However I'd have expected multiple events in that list.

Reply:
Ah thanks, I miss that one.